capture/project_library/project.py

```diff
@@ -72,7 +72,7 @@ def close(self, save: bool) -> None:
         if self.project_cfg.type == "cw":
             self.project.close(save = save)
         elif self.project_cfg.type == "ot_trace_library":
-            self.project.flush_to_disk()
+            self.project.close(save = save)
```

Comment:
Nit: now that the handling of the two databases is more aligned, the `if / else if` construct could be dropped.